packages/react-ds/src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useEffect, useCallback } from 'react'
+
+import { IPagination } from '@venice/core/models'
+import styles from '@venice/styles/components/Pagination.module.scss'
+
+import {
+  getRangeIndexes,
+  hasNextRange,
+  hasPreviousRange,
+} from './helpers/PaginationHelper'
+// TODO: these icons should be replaced when there is an icon library
+import AngleLeft from './icons/AngleLeft'
+import AngleRight from './icons/AngleRight'
+import PaginationRange from './PaginationRange'
+
+const Pagination: React.FC<IPagination> = ({
+  count = 1,
+  page = 1,
+  isLoading = false,
+  onChange = () => null,
+}: IPagination) => {
+  const [isMobile, setIsMobile] = useState(false)
+
+  useEffect(() => {
```

Comment:
I suggest to this entire useEffect block, separate concerns by create a method thats receive a `maxPhoneWidht` and returns if is mobile or not (boolean), and on `useEffect`, you can add listner